joint_offsets:
  joint_1: 0.0
  joint_2: 0.0
  joint_3: 0.0
  joint_4: -0.0
  joint_5: 0.0
  joint_6: 0.0

  joint_7: 0.0
  joint_8: 0.0
  joint_9: 0.0
  joint_10: -0.0
  joint_11: -0.0
  joint_12: 0.0
